storybook/story-components/insert-between.js

```diff
@@ -0,0 +1,9 @@
+const insertBetween = (item, array) =>
+  array.reduce((acc, curr, i, { length }) => {
+    if (i && i < length) {
+      return [...acc, item(), curr];
+    }
+    return [...acc, curr];
+  }, []);
```

Comment:
Not sure I understand why we have this?

Is it to showcase testing?

Otherwise `map`, `Fragment`, and `index` check keeps a viewer from having to context switch 👍

Reply:
Yeah, this is for displaying stories in storybook. Those are wrapping components and `insert-between.js` used to divide multiple components in story inside `DocText` view.